client-app/src/components/AvailabiityRooms/AvailabilityRooms.js

```diff
@@ -0,0 +1,167 @@
+import {roomData} from "../../data/room_data";
+import React, {useState} from "react";
+import "./styles.scss"
+import {FaCheck, FaPlus, FaUserAlt, FaWindowMinimize} from "react-icons/fa";
+import {Button, Dropdown, DropdownItem, DropdownMenu, DropdownToggle} from "reactstrap";
+import {bed_types} from "../../constants/constants";
+import roomFacilitiesData from "../../data/room_facilities";
+
+function AvailabilityRooms(props) {
+
```

Comment:
Please add a line to read the path variable mentioned earlier in the app.js file